modules/apps/27-interchain-accounts/controller/types/msgs.go

```diff
@@ -46,3 +48,24 @@ func (msg MsgRegisterAccount) GetSigners() []sdk.AccAddress {
 
 	return []sdk.AccAddress{accAddr}
 }
+
+// NewMsgSubmitTx creates a new instance of MsgSubmitTx
+func NewMsgSubmitTx(connectionID, owner string, timeoutHeight clienttypes.Height, timeoutTimestamp uint64, msgs []*icatypes.InterchainAccountPacketData) *MsgSubmitTx {
+	return &MsgSubmitTx{
+		ConnectionId:     connectionID,
+		Owner:            owner,
```

Comment:
This should be an AccAddress I believe because it is on the controller chain, we know it should be a valid address

Reply:
is it better to cast it in the proto or check if its a valid addr in ValidateBasic?